src/Matterhorn/State/Channels.hs

```diff
@@ -1083,3 +1084,26 @@ updateChannelNotifyProps cId notifyProps = do
             Just tId -> mh $ invalidateCacheEntry $ ChannelSidebar tId
 
         csChannel(cId).ccInfo.cdNotifyProps .= notifyProps
+
+toggleChannelFavoriteStatus :: MH()
```

Comment:
This is a style note: the type signature here should be `MH ()`, with a space between `MH` and `()`.

Reply:
Done.